boards/px4/fmu-v5/default.cmake

```diff
@@ -18,6 +18,7 @@ px4_add_board(
 	DRIVERS
 		adc
 		barometer # all available barometer drivers
+        rotor_frequency
```

Comment:
```suggestion
		rotor_frequency
```

Can you keep the list sorted alphabetically?

Reply:
Sure, repaired in new commit